### ISSUE TITLE
E3001 Invalid resource attribute Description for resource types AWS::IAM::Role and AWS::IAM::ManagedPolicy

### ISSUE THREAD

**Issue by raphabot** [aws-cloudformation/cfn-lint#2641]
### CloudFormation Lint Version

cfn-lint 0.75.1

### What operating system are you using?

Ubuntu

### Describe the bug

Whenever I lint a template that contains either a resource of type `AWS::IAM::ManagedPolicy` or `AWS::IAM::Role`, if that resource type contains the attribute `Description`, which is defined in both resources documentation[1][2], it returns an `E3001 Invalid resource attribute Description for resource ...`.

[1] https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-managedpolicy.html#cfn-iam-managedpolicy-description
[2] https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html#cfn-iam-role-description

### Expected behavior

I expect to such resources don't trigger rule E3001.

### Reproduction template

```yaml
AWSTemplateFormatVersion: "2010-09-09"
Description: cross account IAM role info that customer needs to create
Parameters:
  ExternalID:
    Type: String
    Description: Your Cloud One Account ID
  NSaaSAccountId:
    Type: String
    Description: Cloud One Network Security NSaaS Account ID
    Default: "852329647234"
  NetworkSecurityAwsAccountId:
    Type: String
    Description: Cloud One Network Security Account ID
    Default: "737318609257"
Resources:
  NetworkSecurityPolicy:
    Type: AWS::IAM::ManagedPolicy
    Description: Network Security Custom Policy
    Properties:
      ManagedPolicyName: NetworkSecurityPolicy
      PolicyDocument:
        Version: "2012-10-17"
        Statement:
          - Sid : cloudconnectorEc2
            Effect: Allow
            Action:
              - "ec2:CreateSubnet"
              - "ec2:CreateTags"
              - "ec2:CreateVpcEndpoint"
              - "ec2:DeleteSubnet"
              - "ec2:DeleteVpcEndpoints"
              - "ec2:DescribeImages"
              - "ec2:DescribeInternetGateways"
              - "ec2:DescribeInstances"
              - "ec2:DescribeNetworkInterfaces"
              - "ec2:DescribeAvailabilityZones"
              - "ec2:DescribeVpcs"
              - "ec2:DescribeRegions"
              - "ec2:DescribeNatGateways"
              - "ec2:DescribeSubnets"
              - "ec2:DescribeKeyPairs"
              - "ec2:DescribeRouteTables"
              - "ec2:DescribeSecurityGroups"
              - "ec2:DescribeVpcEndpoints"
              - "logs:CreateLogGroup"
              - "logs:CreateLogStream"
              - "logs:PutLogEvents"
              - "logs:DescribeLogStreams"
            Resource: '*'
          - Sid: cloudconnectorCF
            Effect: Allow
            Action:
              - "cloudformation:CreateStack"
              - "cloudformation:DeleteStack"
              - "cloudformation:DescribeStacks"
              - "cloudformation:DescribeStackEvents"
              - "cloudformation:ListStacks"
            Resource: '*'
          - Sid: cloudconnectorElb
            Effect: Allow
            Action:
              - "elasticloadbalancing:DescribeLoadBalancers"
              - "elasticloadbalancing:DescribeTargetGroups"
              - "elasticloadbalancing:DescribeTargetHealth"
            Resource: '*'
          - Sid: cloudconnectorIamPolicy
            Effect: Allow
            Action:
              - "iam:GetPolicyVersion"
              - "iam:GetPolicy"
            Resource: "arn:aws:iam::*:policy/NetworkSecurityPolicy"
          - Sid: cloudconnectorIamRole
            Effect: Allow
            Action:
              - "iam:GetRole"
              - "iam:ListAttachedRolePolicies"
            Resource: "arn:aws:iam::*:role/NetworkSecurityRole"
  NetworkSecurityRole:
    Type: AWS::IAM::Role
    Description: Network Security Role
    Properties:
      RoleName: NetworkSecurityRole
      AssumeRolePolicyDocument:
        Version: "2012-10-17"
        Statement:
          - Effect: Allow
            Action: "sts:AssumeRole"
            Principal:
              AWS:
                - !Ref NetworkSecurityAwsAccountId
                - !Ref NSaaSAccountId
            Condition:
              StringEquals:
                sts:ExternalId: !Ref ExternalID
      ManagedPolicyArns:
        - !Ref NetworkSecurityPolicy
```
